docs/dev/netcore.md

```diff
@@ -40,4 +40,27 @@ sudo dpkg -i ~/dotnetpackages/dotnet-sharedframework-ubuntu-x64.1.0.0-rc2-300241
 sudo dpkg -i ~/dotnetpackages/dotnet-sdk-ubuntu-x64.1.0.0-rc2-002416.deb
 ```
 
-> <sub><sup>TIP: Uninstall any existing version first by listing existing packages containing "dotnet" in the name with "dpkg --get-selections | grep dotnet" and then uninstall one by one with "sudo apt-get purge dotnet_package_name".</sup></sub>
\ No newline at end of file
+> <sub><sup>TIP: Uninstall any existing version first by listing existing packages containing "dotnet" in the name with "dpkg --get-selections | grep dotnet" and then uninstall one by one with "sudo apt-get purge dotnet_package_name".</sup></sub>
+
+## ![Redhat](../redhat.png) Red Hat Enterprise Linux 7.2
```

Comment:
It is a different URL to download the .Net Core for Centos, but otherwise it is the same.